entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/PlaceImpl.java

```diff
@@ -164,10 +164,6 @@ public void setDcTermsHasPart(Map<String, List<String>> dcTermsHasPart) {
 		return tmpIsPartOf;
 	}
 
-	@Override
-	public String getType() {
-		return "Place";
```

Comment:
please revert back this code, and use EntityTypes.Place.getEntityType() instead of the hardcoded value

Reply:
Done.